code/modules/projectiles/guns/specialist.dm

```diff
@@ -220,6 +226,26 @@
 	damage_mult = BASE_BULLET_DAMAGE_MULT
 	recoil = RECOIL_AMOUNT_TIER_5
 
+/obj/item/weapon/gun/rifle/sniper/M42A/co
+	name = "\improper M42AC scoped rifle"
+	desc = "A heavy sniper rifle manufactured by Armat Systems. It has a scope system and fires armor penetrating rounds out of a 15-round magazine. This one comes in Regal Blue, indicating it belongs to a commanding officer. An inscription reads: 'Out of Sight, Into Mind'"
+	icon_state = "m42ac"
+	item_state = "m42ac"
+	map_specific_decoration = FALSE
+	zoom_vision_post_update = TRUE
+
+/obj/item/weapon/gun/rifle/sniper/M42A/co/skill_check(var/mob/living/user)
+	if(user.job == GET_DEFAULT_ROLE(JOB_CO))
+		return TRUE
+	return ..()
+
+/obj/item/weapon/gun/rifle/sniper/M42A/co/set_bullet_traits()
+	return
+
+/obj/item/weapon/gun/rifle/sniper/M42A/co/handle_zoom_vision(var/mob/living/carbon/human/H)
+	H.see_in_dark = 12
+	H.lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
```

Comment:
why is this on the gun not the scope 💀 

also I don't get this feature, really. It makes it better to look down your rifle scope to get binoc vision than down actual binocs, which is weird. 

The lack of iff also makes it somewhat useless other than when - I guess - pushing a wide open area?

because snipers suck in fob defence (can't finish off kills)